setup.py

```diff
@@ -49,7 +49,7 @@
 from utils.command_style import source_style
 from utils.command_clean import source_clean
 from utils.command_build_clib import custom_build_clib
-from utils.dpnp_build_utils import find_mathlib
+from utils.dpnp_build_utils import find_cmplr, find_mathlib, find_omp, IS_LIN, IS_WIN
```

Comment:
moving IS_LIN or IS_WIN logic to `utils` is completely wrong. In utils we have to have simple procedure to find library (headers) by paths. Nothing else.